tasmota/my_user_config.h

```diff
@@ -395,7 +395,9 @@
                                                  // Enabled by default on ESP32 and variants
 
 // -- ESP-NOW -------------------------------------
-//#define USE_TASMESH                              // Enable Tasmota Mesh using ESP-NOW (+11k code)
+#define USE_TASMESH                              // Enable Tasmota Mesh using ESP-NOW (+11k code)
+#define TASMESH_HEARTBEAT      1                 // If enabled, the broker will detect when nodes come online and offline and send Birth and LWT messages over MQTT correspondingly (0 = off, 1 = on)
```

Comment:
We usually take the convention of the feature to be defined instead of having value 1. Also prefix with `USE_`

Should be `#define USE_TASMESH_HEARTBEAT`